.github/workflows/main-CI.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: '3.7'
+          python-version: '3.8'
```

Comment:
Is the theory here that the actual `foursight-<portal>` repos will test this enough that we don't need to do all versions? Do the tests for this take a long time? I otherwise think this should maybe have a matrix of 3.8, 3.9, 3.10, 3.11. Thoughts, @willronchetti ?